llvm.spec

```
@@ -79,5 +79,8 @@ rm -f %{i}/bin/FileRadar.scpt %{i}/bin/GetRadarVersion.scpt
 # Avoid dependency on /usr/bin/python, Darwin + Xcode specific
 rm -f %{i}/bin/set-xcode-analyzer
 
+# Remove all static libs except libomptarget-nvptx.a
+find %{i}/lib64 -name "*.a" -and -not -name "libomptarget-nvptx.a" | xargs rm -f
```

```suggestion
find %{i}/lib64 -name "*.a" -and -not -name "libomptarget-*.a" -delete
```

This should be slightly safer:
  - `find -delete` should handle any non-printable characters and race conditions better than `xargs`
  - `libomptarget-*.a` is more future-proof, in case new backends are added